### ISSUE TITLE
Does the diversity of model outputs have an impact on the metric score?

### ISSUE THREAD

**Issue by 27182812** [salesforce/DialogStudio#4]
Hello, excellent work! I'm curious if the diversity of the model outputs will have an effect on the metric scores. Specifically, the baselines in the paper generate responses based on the context under the settings of zero-shot and few-shot, but the outputs of the models will be different when running multiple times, and whether this will affect the stability of the evaluation results.

**Comment by jianguoz** [comment]
Hi @27182812, thanks for your interest in the work:) 

Baselines & Our Model: For both the baseline models and our own, we've adhered to the recommended best settings.

Hyper-parameters: We've stuck with the default settings when it comes to hyper-parameters like top_p, temperature, max_length, and min_length. During inference, we use model.generate(input_ids) and I didn't observe significant variations between the models.  By the way, I just uploaded the version 1.0 models: https://github.com/salesforce/DialogStudio#model. 

Templates & Formatting: When it comes to specific formatting like the order of (instructions, dialogue input, knowledge), it plays a pivotal role, especially in zero-shot scenarios. I noticed a substantial drop in performance for models like GODEL and FLAN-T5 if we do not strictly follow their specific formats and special tokens. Consequently, we've been meticulous about following their formats to achieve optimal performance. I hypothesize that such issues might be less prominent with larger models (>=7B).

I hope this provides a clearer picture for you:)
